packages/web-components/@sfx/autocomplete/src/autocomplete.ts

```diff
@@ -26,6 +30,7 @@ export default class Autocomplete extends LitElement {
   constructor() {
     super();
     this.receivedResults = this.receivedResults.bind(this);
+    this.handleHoverTerm = this.handleHoverTerm.bind(this);
```

Comment:
This no longer needs to be bound here if using `@` in the [render method](https://github.com/groupby/sfx-view/pull/31/files#diff-15c2226c616cd93d909210ac1585dd59R101)